packages/stencil/src/components/datacom-avataravatar/datacom-avataravatar.tsx

```diff
@@ -0,0 +1,50 @@
+import { Component, Host, Prop, h } from '@stencil/core';
+
+@Component({
+  tag: 'datacom-avataravatar',
+  styleUrl: 'datacom-avataravatar.css',
+  shadow: true,
+})
+export class DatacomAvataravatar {
+  /* Custom properties */
+  @Prop() firstName: string;
+  @Prop() lastName: string;
+  @Prop() jobTitle: string;
+  @Prop() companyName: string;
+  @Prop() src: string;
+
+  render() {
+    let avtImage;
+    let F;
```

Comment:
As commented by Andrew, you should define some better variable names here. Something more descriptive so other developers know what these are meant to represent at a glance. (remember to use camelCase too!)